module/VuFind/src/VuFind/ILS/Driver/SierraRest.php

```diff
@@ -684,16 +684,14 @@ public function patronLogin($username, $password)
             $lastname = $parts[0];
             $firstname = $parts[1] ?? '';
         }
-        return [
-            'id' => $patron['id'],
-            'firstname' => $firstname,
-            'lastname' => $lastname,
-            'cat_username' => $username,
-            'cat_password' => $password,
-            'email' => !empty($patron['emails']) ? $patron['emails'][0] : '',
-            'major' => null,
-            'college' => null,
-        ];
+        return $this->createPatronArray(
+            id: $patron['id'],
+            firstname: $firstname,
+            lastname: $lastname,
+            cat_username: $username,
+            cat_password: $password,
+            email: $patron['emails'][0] ?? ''
```

Comment:
Not sure it matters, but would it make more sense for a missing email to default to `null` rather than `''`?